File: docs/content/guides/snippets/aws-lambda/lambda.py
import os

import requests


def handler(event, context):
    token = os.environ.get("GITHUB_API_TOKEN")
    url = "https://api.github.com/repos/dagger/dagger/issues"
    headers = {
        "Accept": "application/vnd.github+json",
        "Authorization": f"Bearer {token}",
    }
    r = requests.get(url, headers=headers)
    return r.json()

File: docs/current_docs/guides/snippets/aws-lambda/lambda.js
import("node-fetch")

async function handler(event) {
  const token = process.env["GITHUB_API_TOKEN"]
  const headers = {
    Accept: "application/vnd.github+json",
    Authorization: `Bearer ${token}`,
  }
  const response = await fetch(
    "https://api.github.com/repos/dagger/dagger/issues",
    {
      headers: headers,
    },
  )
  const data = await response.json()
  return data
}

module.exports.handler = handler
